### PR TITLE
Abb 195

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Sorry, this might have been superfluous.

### CODE REVIEW

File: ParisBNF/et/BNFet150.xml [hunk 1]
```diff
@@ -70,284 +70,284 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                               <title type="complete">Names of the sun and some stars, of idols, terms of
                     government, names of currencies and weights</title>
                               <textLang mainLang="gez" otherLangs="am"/>
                            </msItem>
-                  
+
                            <msItem xml:id="p1_i2.2.12">
                               <locus from="41r" facs="f85"/>
-                              <title type="complete">On the apostles and their families</title>
+                              <title type="complete" corresp="LIT5849GenealogyApostles">On the Apostles and their families</title>
```
Comment (eu-genia) [inline ParisBNF/et/BNFet150.xml:265]: should be ref, not corresp. I think

Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet150.xml:265]: corrected

File: ParisBNF/abb/BNFabb195.xml [hunk 2]
```diff
@@ -0,0 +1,683 @@
...
+                    <term key="ChristianLiterature"/>
+                    <term key="OldTestament"/>
+                    <term key="Religion"/>
+                    <term key="ZaMa"/>
+                  </keywords>
+            </textClass>
+            <langUsage><language ident="gez">Gǝʿǝz</language></langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2019-12-12">Created entity</change>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:675]: corrected

File: ParisBNF/abb/BNFabb195.xml [hunk 3]
```diff
@@ -0,0 +1,683 @@
...
+
+                          <msItem xml:id="ms_i7.2">
+                          <locus from="226ra" to="228vb"/>
+                          <title type="complete" ref="LIT1202Bookof"/>
+                           <incipit xml:lang="gez"><hi rend="rubric"/>
+                           <hi rend="rubric">ዝነገረ᎓ መጽሐፍ᎓ ዘጸሐፈ᎓ ባሮክ᎓ ወልደ᎓ ኔርዩ᎓ ወልደ፡ ማሲው፡ ወልደ፡</hi>ሴደቅዮ፡ ባቢሎን፡
+                         በኃምስ፡ ዓመት፡</incipit>
+                              <explicit xml:lang="gez">በፍሥሐ᎓ ወበብርሃነ᎓ ስብሐቲሁ᎓ በምሕረቱ፡ ወበጽድቁ፨
+                          </explicit>
+                          <note>Book of Baruch.</note>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb195.xml:252]: I know that you like these `<note`>s with the title, but I really feel that they add no information and contribute to making the page more cluttered...

File: ParisBNF/abb/BNFabb195.xml [hunk 4]
```diff
@@ -0,0 +1,683 @@
...
+            </layoutDesc>
+              </objectDesc>
+
+                <handDesc>
+              <handNote script="Ethiopic" xml:id="h1" corresp="#ms_i1 #ms_i2
+            #ms_i3 #ms_i4 #ms_i5 #ms_i6 #ms_i7 #ms_i8 #a2">
+                  <date notBefore="1800" notAfter="1840"/>
+                  <seg type="ink">Black, red (light vermillion)</seg>
+                  <desc>Written in a skilled but irregular,
+                    hasty hand. Small script, ca. 20-22 signs per line.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb195.xml:419]: out of curiosity, why signs and not characters?

File: ParisBNF/abb/BNFabb195.xml [hunk 5]
```diff
@@ -0,0 +1,683 @@
...
+                            above.</desc>
+                        </decoNote>
+                      </decoDesc>
+
+                        <additions>
+                          <list>
+                            <item xml:id="a1">
+                              <locus target="#1r"/>
+                               <desc type="OwnershipNote" xml:lang="gez">Written in a very crude hand, mentioning an owner
+                               <persName ref="PRS12936WaldaSamaet">Walda Samāʿǝt of ʾAmhāra</persName></desc>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb195.xml:483]: Samāʿt

File: ParisBNF/abb/BNFabb195.xml [hunk 6]
```diff
@@ -0,0 +1,683 @@
...
+
+                    <history>
+                        <origin>
+                     <origDate when="1649" evidence="internal-date"/>
+                     <note>The writing of the manuscript was finished around <date>1649</date>,
+                     according to the date mentioned in <ref target="#coloph1"/>.</note>
+                     </origin>
+                       <provenance>The manuscript was owned by
+                         <persName ref="PRS9683Urbino">
+                           abba Yostos</persName> who donated it to <persName ref="PRS1071dAbbadi">Antoine d'Abbadie</persName>
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb195.xml:592]: ʾabbā Yosṭos

File: ParisBNF/et/BNFet144.xml [hunk 7]
```diff
@@ -93,10 +93,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <textLang mainLang="am" otherLangs="gez"/>
                         <note>The first part is written in Amharic, the second in Gǝʿǝz.</note>
                      </msItem>
-              
+
                      <msItem xml:id="ms_i1.6">
                         <locus from="10v" to="11r" facs="f26"/>
-                        <title type="complete" ref="LIT2422Tarika#GenealogyApostels">Genealogy of the twelve apostles</title>
+                        <title type="complete" ref="LIT5849GenealogyApostles">Genealogy of the twelve apostles</title>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/et/BNFet144.xml:99]: It has been updated.

File: ParisBNF/abb/BNFabb195.xml [hunk 8]
```diff
@@ -0,0 +1,683 @@
...
+                            </incipit>
+                            <explicit xml:lang="gez">ተፈጸመ፡ ኢሳይያስ፡ ወልደ፡ አሞጽ፡ ምስለ፡ ዕርገቱ፨</explicit>
+                            <colophon xml:id="coloph1">ዝንቱ፡ መጽሐፍ፡ ተጽሕፈ፡ በ<date calendar="ethiopian" when="1649"
+                               when-custom="7141">፸፻ወ፩ወ፻፡ ዓመተ፡ ምህረት፡ ወሠረቀ፡ ሌሊትኒ፡
+                              ፰ወመዓልትኒ፡ ፳</date>፨
+                            </colophon>
+                            </msItem>
+                            <msItem xml:id="ms_i6">
+                             <locus from="88ra" to="140vb"/>
+                             <title type="incomplete" ref="LIT1315Dodeka"/>
```
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:155]: It is.

File: ParisBNF/abb/BNFabb195.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:252]: Yes, of course, I will remove. It is mostly for me, for orientation, as the IDs frequently give no hint as to the text they denote ("Bookof"). As my mind is not digital and needs some concret points to grasp, I have to assist myself :) I will remove them at the end.

File: ParisBNF/abb/BNFabb195.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:419]: No special reason, but I can change if you think "character" is prevailing in the DB and should be preferably used for consistency.

File: ParisBNF/abb/BNFabb195.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:483]: ok

File: ParisBNF/abb/BNFabb195.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:592]: Ok. Thanks!

File: ParisBNF/abb/BNFabb195.xml [hunk 9]
```diff
@@ -672,7 +674,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <langUsage><language ident="gez">Gǝʿǝz</language></langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="DN" when="2019-12-12">Created entity</change>
+            <change who="DN" when="2020-05-12">Created entity</change>
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb195.xml:677]: I think we have August, not that it is so important

I would also add a change element to the edited records

Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:677]: ok, i'll do

File: ParisBNF/abb/BNFabb195.xml (hunk 4 above)
Comment (thea-m) [reply ParisBNF/abb/BNFabb195.xml:419]: It's just what we used for DS. Later in this record, there is "letters". I was wondering whether there might be a conceptual difference (counting or not word separator or so).

File: ParisBNF/abb/BNFabb195.xml (hunk 3 above)
Comment (thea-m) [reply ParisBNF/abb/BNFabb195.xml:252]: ah, I see

File: ParisBNF/abb/BNFabb195.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb195.xml:419]: There might be; I used to use one general term for all characters applied in the entire Ethiopic writing system (including word dividers and etc., when we count characters per line, we count everything), while letters refer only to letters, numerals to numerals etc.

**Pull request merged**